ReactAndroid/src/main/java/com/facebook/react/modules/websocket/WebSocketModule.java

```diff
@@ -251,8 +241,7 @@ public void ping(int id) {
     }
     try {
       Buffer buffer = new Buffer();
-      client.sendPing(buffer);
```

Comment:
It's not removed yet right? Let's add it to avoid a breaking change. We'll figure it out later.